### ISSUE TITLE
Simplify Secrets Management [Placeholder]

### ISSUE THREAD

**Issue by rotated8** [emory-libraries/blacklight-catalog#686]
Currently, to deploy one needs an up-to-date dotenv with all our secrets. It would reduce the friction for people to deploy if that was only necessary when there were changes to the secrets.

**Comment by bwatson78** [comment]
@rotated8 On July 8th, `40a0f0e6c642d1f5723dc75ef0cac105a30f5623` was merged into main and has been a part of multiple versions. That commit scrapes the ENV keys/values from a YAML file housed on each environment's server (`/opt/blacklight-catalog/shared/config/environment_variables.yml`) at the start of each deploy. 

Like we discussed earlier, further work is needed to encrypt this file into the Terraform/Ansible code used to rebuild the servers and application when we restore from backups (or for other reasons).

I have also placed the current keys/values of all environments into the shared [OneDrive SE folder](https://emory-my.sharepoint.com/personal/cbrittl_emory_edu/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fcbrittl%5Femory%5Fedu%2FDocuments%2FBox%2FLTDS%2DSoftware%20Engineering&FolderCTID=0x01200074C645A8322CCC4AB0DAE8EB787D4119) as a one-stop reference point. I will post this to the SE chatroom for easy reference.
